test/e2e_node/image_list.go

```diff
@@ -106,7 +106,7 @@ func isRunningOnArm64() bool {
 }
 
 func getNodeProblemDetectorImage() string {
-	const defaultImage string = "registry.k8s.io/node-problem-detector/node-problem-detector:v0.8.7"
```

Comment:
let's add it to build/dependencies.yaml and bump this one as well: https://github.com/kubernetes/kubernetes/blob/b3d94ae74f3284b64a32a9fa2d82509c86d22e17/cluster/gce/gci/configure.sh#L29

Reply:
Added entry in build/dependencies.yaml and left a TODO for myself to ensure new dependencies get updated to a couple of places (from where we pull them down)

Reply:
(see above!)